src/assisted_test_infra/download_logs/download_logs.py

```diff
@@ -164,6 +164,12 @@ def download_logs(
             client.download_cluster_events(cluster["id"], get_cluster_events_path(cluster, output_folder))
             shutil.copy2(os.path.join(os.path.dirname(os.path.realpath(__file__)), "events.html"), output_folder)
 
+        infra_env_list = client.get_infra_env_by_cluster_id(cluster["id"])
```

Comment:
It's not better to iterate over the hosts infra_envs?

Reply:
I am getting all infra envs related to certain cluster - so i think it covers them all, no?

Reply:
done